integration/integration_test.go

```diff
@@ -30,6 +30,8 @@ var (
 	image = flag.String("image", "", "gadget container image")
 
 	doNotDeploy = flag.Bool("no-deploy", false, "don't deploy Inspektor Gadget")
+
+	deploySPOFlag = flag.Bool("deploy-spo", true, "deploy the Security Profiles Operator (SPO)")
```

Comment:
What about following the same approach for both and making it `no-deploy-spo`?

Reply:
Right, updated to follow this approach.